hw9_task_2.py

```diff
@@ -0,0 +1,77 @@
+class Staff():
+
+    def __init__(self, name=None, gender=None, age=None, job_position=None, years_experience=None):
+        self.__name = name
+        self.__gender = gender
+        self.__age = age
+        self.__job_position = job_position
+        self.__years_experience = years_experience
+
+    @property
+    def name(self):
+        return self.__name
+
+    @property
+    def gender(self):
+        return self.__gender
+
+    @property
+    def age(self):
+        return self.__age
+
+    @property
+    def job_position(self):
+        return self.__job_position
+
+    @property
+    def years_experience(self):
+        return self.__years_experience
+
+    @name.setter
+    def name(self, staff_name):
+        if isinstance(staff_name, str):
+            self.__name = staff_name
+        else:
+            raise TypeError
+
+    @gender.setter
+    def gender(self, staff_gender):
+        if isinstance(staff_gender, str) and staff_gender == 'male' or staff_gender == 'female':
```

Comment:
it can be shorter if you use list. 
You can create constant GENDER = ['male', 'female']
and check here if staff_gender in GENDER